src/book/03-guides/03-hardware/02-servo-calibration.mdx

```diff
@@ -0,0 +1,125 @@
+---
+section: Guides
+chapter: Hardware
+title: Servo Setup and Calibration
+description: Learn how to test and calibrate servos.
+slug: /guides/hardware/servo-calibration
+---
+
+Servos are electromechanical components that allow for the precise control and positioning of the various limbs attached to the NUgus robot. Usually, when a servo is found to not be performing correctly (i.e. seizing or a crunching sound can be heard), it should be removed and replaced with a new servo.
+
+Before a servo is first used, its default parameters need to be updated to ensure correct operation. This is done using [Dynamixel Wizard](https://www.robotis.com/service/download.php?no=1670).
```

Comment:
Please make it clearer that clicking on this link starts the download and perhaps mention that it's a windows exe

Reply:
Reworded